lib/chef/knife/ec2_server_create.rb

```diff
@@ -817,6 +843,7 @@ def bootstrap_for_windows_node(server, fqdn)
         bootstrap.config[:install_as_service] = locate_config_value(:install_as_service)
         bootstrap.config[:session_timeout] = locate_config_value(:session_timeout)
         bootstrap.config[:tags] = config[:tags] if locate_config_value(:tag_node_in_chef)
+        bootstrap.config[:tags] = config[:chef_tag] if locate_config_value(:chef_tag)
```

Comment:
This looks to be a repetition of code. we done same thing for linux node. Move this to common method.